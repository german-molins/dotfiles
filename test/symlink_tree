#!/usr/bin/bash

[ -e dest ] && rm -r dest

. ../symlink.sh
create_symlinks .symlink source/ dest/
if cmp -s symlink_tree.out <(diff -rq source/ dest/)
then
	echo 'test successful'
	exit 0
else
	echo 'test failed'
	exit 1
fi
