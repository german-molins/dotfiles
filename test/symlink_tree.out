Only in source/d1/d2: .symlink
Only in source/d1: .symlink
Only in source/d2/d1: .symlink
Only in source/d2: f1
Only in source/d2: f2
Only in source/: .symlink
